emcc.py

```diff
@@ -1281,6 +1282,18 @@ def check(input_file):
       shared.Settings.DECLARE_ASM_MODULE_EXPORTS = 1
       logger.warning('Enabling -s DECLARE_ASM_MODULE_EXPORTS=1, since MODULARIZE currently requires declaring asm.js/wasm module exports in full')
 
+    if shared.Settings.MINIMAL_RUNTIME:
+      # Minimal runtime uses a different default shell file
+      if options.shell_path == shared.path_from_root('src', 'shell.html'):
+        options.shell_path = shared.path_from_root('src', 'shell_minimal_runtime.html')
```

Comment:
A new default shell file for MINIMAL_RUNTIME shows a simple example of how the user should XHR in the compiled code files. The model of how asm.js/wasm/mem file glues together with the main JS file stays very much the same.